commander/package.json

```diff
@@ -20,7 +20,7 @@
 		"url": "https://github.com/LiskHQ/lisk-sdk/issues"
 	},
 	"engines": {
-		"node": ">=18.12.0 <=18",
+		"node": ">=18.19.1 <=18",
```

Comment:
no need to change the min version so you can choose

Reply:
only for commander or for all? then in that case we don't even need this PR

Reply:
for all the package.json =)